### ISSUE TITLE
AVT 1 - Tile Multi-Select: DAP violation 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon-components-react#2327]
Environment
macOS Mojave version 10.14.4
Chrome Version 74.0.3729.131 (Official Build) (64-bit)

Detailed Description
Run DAP with Feb. 2019 ruleset on the Tile Multi-select component
See screenshot below for violations related to missing group and label:
![multi-select](https://user-images.githubusercontent.com/21676914/57316572-baff4e80-70bb-11e9-8b22-9ca03fc32df3.png)

As shown in the screenshot adding an ARIA role="group" and aria-label="selectable tiles"  fixed the issue.

See related issue #2238
